e2e/_suites/ingest-manager/features/fleet_mode_agent.feature

```diff
@@ -6,17 +6,9 @@ Feature: Fleet Mode Agent
 Scenario Outline: Deploying the <os> agent
   When a "<os>" agent is deployed to Fleet
   Then the agent is listed in Fleet as "online"
-    And system package dashboards are listed in Fleet
-Examples:
-| os     |
-| centos |
-| debian |
-
-@start-agent
-Scenario Outline: Starting the <os> agent starts backend processes
-  When a "<os>" agent is deployed to Fleet
-  Then the "filebeat" process is in the "started" state on the host
+    And the "filebeat" process is in the "started" state on the host
```

Comment:
What do you think of adding `And the "elastic-agent" process is in the "started" state on the host` as first check?

Reply:
love it. good idea!

Reply:
Super NIT: What about reordering the steps in terms of importance or order of appearance?

```gherkin
Then the "elastic-agent" process is in the "started" state on the host
    And the "filebeat" process is in the "started" state on the host
    And the "metricbeat" process is in the "started" state on the host
    And the agent is listed in Fleet as "online"
    And system package dashboards are listed in Fleet
```

Reply:
@EricDavisX; i'm gonna merge and send a PR with the comments above.

Thanks for your contribution!!